pkg/cni/kubeconfig.go

```diff
@@ -114,7 +114,6 @@ func maybeWriteKubeConfigFile(serviceAccountPath, kubeconfigFilepath string) err
 	}
 
 	if shouldCreateKubeConfigFile(kc, kubeconfigFilepath) {
-		log.Info("kubeconfig either does not exist or is out of date, writing a new one")
```

Comment:
Still keep the log? Make sure kubeconfig is really updated, changing it to `Debug` maybe enough.

Reply:
There is already a similar log after calling maybeWriteKubeConfigFile